internal/schema-registry/command_schema_compatibility_validate.go

```diff
@@ -21,25 +21,24 @@ type validateOut struct {
 
 func (c *command) newSchemaCompatibilityValidateCommand(cfg *config.Config) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "validate",
+		Use:   "validate <schema-file>",
```

Comment:
Nit: We support full filepaths here, right?
```suggestion
		Use:   "validate <schema-path>",
```